src/setupWorker/glossary.ts

```diff
@@ -54,6 +59,14 @@ export type StartOptions = SharedOptions & {
    * instance is ready. Defaults to `true`.
    */
   waitUntilReady?: boolean
+
+  /**
+   * A predicate function that allows you override the default strict matching behavior.
```

Comment:
People are most likely not familiar with what a "strict matching behavior" is, as it references to some internal logic.

I suggest we give a short description over what this method does:

```
A custom lookup function to find a Mock Service Worker in the list of all registered Service Workers on the page.
```

I'd also drop a usage suggestions and rather include them in the documentation for this option. What do you think?

Reply:
Agreed. Will open another PR in the docs repo shortly.

Reply:
Thanks! Don't rush with the docs, let's merge this first :)

Reply:
The docs need to be ready when we release this, not merge. It's perfectly fine to write them after, once we underwent the review process.